native-schema-locater/src/SchemaXmlFileLocater.ts

```diff
@@ -23,6 +25,13 @@ function isECv2Schema(schemaText: string): boolean {
  * @internal This is a workaround the current lack of a full xml parser.
  */
 export class SchemaXmlFileLocater extends SchemaFileLocater implements ISchemaLocater {
+
+  constructor() {
+    super();
+    Logger.initializeToConsole();

```

Comment:
Do we always need to show these additional logs? or we should set some flag to enable this logging when required?

Reply:
The log level is set to Error, which I think we would always want to see.  Expanding and configuring the logging could be investigated for all the bis-schema-validation tools really.  @ColinKerr, any comment?

Reply:
I would like to always see these logs in the validation builds and in other cases I think they would be helpful but I do not care much.  In general just showing errors should be a problem for anyone so they can be kept on all the time unless we find that it is causing a problem

Reply:
Sounds good!